### DISCUSSION TITLE
还是关于hexo-theme-next中评论区默认文字修改的问题

### DISCUSSION THREAD

**Discussion by FHR2048** [walinejs/waline#1925] [Q&A] [answered]
我尝试用您给出的locale.placeholder: xxxxxxxx
但还是不可行。

我找到了一个正常的页面,[来自www.techgrow.cn](www.techgrow.cn)
源代码中它是这样的:
```
     "locale":{"placeholder":"支持匿名评论啦，若希望及时收到博主的反馈，建议登录评论或者在上方的邮箱输入框留下邮箱地址哦 (๑•̀ㅂ•́)و✧"}
```
这也和官网中提供的方式类似

我该如何实现生成:
`     "locale":{"placeholder":"xxxxxxxx"}`
而不是
`     "locale.placeholder":"xxxxxxxx"`
和
`     "placeholder":"xxxxxxxx"`

我尝试添加类似下面的配置,但都是无效的
```
waline:
...
  locale: {
    placeholder: 'xxxxxx',
  }
...
```

**Comment by FHR2048** [comment]
您好,我刚刚自己解决了,正确的配置项应该是:
```
waline:
...
  locale: {placeholder: xxxxxxxx}
...
```

**Reply by FHR2048** [reply]
locale: {xxxxxxxxx}的设置不能换行

**Comment by lizheming** [comment]
请自行 Google YAML 语法

**Comment by Mister-Hope** [comment, marked as answer]
https://mrhope.site/code/language/yaml/

上面的看不懂还有一个小白版本的： https://mrhope.site/note/innenu/yaml.html
